macOS/DuckDuckGo/Tab/Model/Tab.swift

```diff
@@ -1355,6 +1366,14 @@ extension Tab/*: NavigationResponder*/ { // to be moved to Tab+Navigation.swift
             webView.setDocumentHtml(html)
         }
     }
+
+    func renderingProgressDidChangeRaw(progressEvents: UInt) {
+        // Emit only after first paint event
```

Comment:
Could you explain the `4` in a comment? My guess, without checking WebKit sources, is that it’s the “first paint” event enum value, but that’s just an assumption that would be great to have explained here.